src/app/shared/menu/providers/browse.menu.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+import { Injectable } from '@angular/core';
+import {
+  Observable,
+  of as observableOf,
+} from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { BrowseService } from '../../../core/browse/browse.service';
+import { PaginatedList } from '../../../core/data/paginated-list.model';
+import { RemoteData } from '../../../core/data/remote-data';
+import { BrowseDefinition } from '../../../core/shared/browse-definition.model';
+import { getFirstSucceededRemoteData } from '../../../core/shared/operators';
+import { TextMenuItemModel } from '../menu-item/models/text.model';
+import { MenuItemType } from '../menu-item-type.model';
+import { PartialMenuSection } from '../menu-provider.model';
+import { AbstractExpandableMenuProvider } from './helper-providers/expandable-menu-provider';
+
+/**
+ * Menu provider to create the browse menu sections
+ */
+@Injectable()
+export class BrowseMenuProvider extends AbstractExpandableMenuProvider {
+  constructor(
+    protected browseService: BrowseService,
+  ) {
+    super();
+  }
+
+  getTopSection(): Observable<PartialMenuSection> {
+    return observableOf(
+      {
+        model: {
+          type: MenuItemType.TEXT,
+          text: 'menu.section.browse_global',
+        } as TextMenuItemModel,
+        icon: 'globe',
+        visible: true,
+      },
+    );
+  }
+
+  getSubSections(): Observable<PartialMenuSection[]> {
```

Comment:
Because this is not a "hardcoded" menu, like many others, we may want to add some code comments here (or at the top of the class) to describe that this menu is getting it's "subsection" options from the current browse definitions defined on the backend.